docs/docs/guides/solution-scenarios/disable-console.mdx

```diff
@@ -0,0 +1,85 @@
+---
+title: Prevent users from accessing ZITADEL console
+sidebar_label: Disable ZITADEL Console
+---
+
+ZITADEL does include the ZITADEL Console, which allows you to manage all ZITADEL resources as well as end users to manage their profile information.
+In some use cases you want to prevent users from accessing the console, this could be:
+- User management is integrated into your own app
+- Users login via SSO and should not be able to change their data in the Console
+- Only managers should be able to access Console to manage their users
+
+At the moment it is not possible to simply disable the Console but with a workaround you can prevent users from accessing the Console.
+
+## Self-hosted
+
+If you are running ZITADEL self-hosted we recommend to restrict the access to ZITADEL Console via WAF/reverse-proxy for Non-Manager users
+
+## ZITADEL Cloud (and self-hosted)
+
+### Default redirect url
+
+The Default Redirect URI will be used, if a user calls the login page directly. More specifically, typically a client will initiate login with an auth request.
```

Comment:
```suggestion
The Default Redirect URI will be used, if a user calls the login page directly. More specifically, typically a client will initiate login with an [auth request](https://zitadel.com/docs/apis/openidoauth/authrequest).
```

Reply:
Should be actually linked and explained in https://github.com/zitadel/zitadel/issues/7399